Octokit.Tests/Clients/UserEmailsClientTests.cs

```diff
@@ -19,7 +19,26 @@ public void GetsCorrectUrl()
                 client.GetAll();
 
                 connection.Received(1)
-                    .GetAll<EmailAddress>(Arg.Is<Uri>(u => u.ToString() == "user/emails"));
+                    .GetAll<EmailAddress>(Arg.Is<Uri>(u => u.ToString() == "user/emails"), 
+                    Arg.Is(ApiOptions.None));
+            }
+
+            [Fact]
+            public void GetsCorrectUrlWithApiOptions()
+            {
+                var options = new ApiOptions
+                {
+                    StartPage = 1,
+                    PageCount = 1
+                };
+
+                var connection = Substitute.For<IApiConnection>();
+                var client = new UserEmailsClient(connection);
+
+                client.GetAll(options);
+
+                connection.Received(1)
+                    .GetAll<EmailAddress>(Arg.Is<Uri>(u => u.ToString() == "user/emails"), Arg.Is<ApiOptions>(apiOptions => apiOptions == options));
```

Comment:
Tnx, @devkhan, you're absolutely right, usage of Any is more convenient in this situation. Fixed in ad3b7823.